### PR TITLE
Update Local Setup to make it more consistent to cloud setup.

### PR DESCRIPTION
**Pull request by cpeeyush**
## What this PR changes/adds

Update Local Setup to make it more consistent to cloud setup.

## Why it does that

So that test runs against similar setup and config remains same and relevant for both the environments.

## Linked Issue(s)

Relates to #241 

## Checklist

- [ ] added appropriate tests?
- [x] performed checkstyle check locally?
- [ ] added/updated copyright headers?
- [ ] documented public classes/methods?
- [x] added/updated relevant documentation?
- [ ] added relevant details to the changelog? (_skip with label `no-changelog`_)
- [ ] formatted title correctly?

### CODE REVIEW

**Review by marcgs — APPROVED** [review]

File: deployment/seed-data.sh [hunk 1]
```diff
@@ -2,10 +2,28 @@
...
 
-newman run \
-  --folder "Publish Master Data" \
-  --env-var data_management_url="http://$EDC_HOST:9191/api/v1/data" \
-  --env-var storage_account="$ASSETS_STORAGE_ACCOUNT" \
-  --env-var participant_id="$PARTICIPANT_ID" \
-  --env-var api_key="$API_KEY" \
-  deployment/data/MVD.postman_collection.json
\ No newline at end of file
+function split_by_comma() {
```
Comment (algattik) [inline deployment/seed-data.sh]: `entrypoint.sh` uses a different method to split, would be good to align to make it easier to maintain for non-bash experts

Comment (algattik) [reply deployment/seed-data.sh]: How about similar to `entrypoint.sh` using a single input string `company1:company1assets,company2:company1assets`... (two of the inputs having redundant values `company1`, 2, 3)

The bash code here is pretty complex

File: system-tests/README.md [hunk 2]
```diff
@@ -237,7 +239,7 @@ docker run -i danedmunds/pem-to-jwk:1.2.1 --public --pretty < system-tests/resou
 
 ### Debugging MVD locally
 
-Follow the instructions in the previous sections to run an MVD with a consumer and provider locally using docker-compose.
+Follow the instructions in the previous sections to run an MVD with a consumer(company2) and provider(company1) locally using docker-compose.
```
Comment (algattik) [inline system-tests/README.md]:
```suggestion
Follow the instructions in the previous sections to run an MVD with a consumer (company2) and provider (company1) locally using docker-compose.
```

File: system-tests/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # System tests
 
-The system tests copies a file from a provider to a consumer blob storage account.
+The system tests copies a file from a provider (`company1`) to a consumer (`company2`) blob storage account.
```
Comment (algattik) [inline system-tests/README.md]: grammar "tests copies". In fact should be reformulated since there are other tests too

**Review by algattik — CHANGES_REQUESTED** [review]

File: deployment/seed-data.sh (hunk 1 above)
Comment (cpeeyush) [reply deployment/seed-data.sh]: @algattik using a single input like `company1:company1assets,company2:company1assets...` will cause the issue for cloud deployment seed data step as the same shell script is used by cloud deployment data seeding as well and their EDC_HOST and PARTICIPANT_ID are different.

Comment (cpeeyush) [reply deployment/seed-data.sh]: Updated script to use similar input and delimiter as `entrypoint.sh`.

**Review by algattik — APPROVED** [review]

File: system-tests/README.md (hunk 3 above)
Comment (cpeeyush) [reply system-tests/README.md]: Updated by @algattik. Thanks for the help. Resolving it.

**Pull request merged**